ui/src/app/applications/components/applications-list/applications-tiles.tsx

```diff
@@ -102,158 +99,180 @@ export const ApplicationTiles = ({applications, syncApplication, refreshApplicat
+                                            className={`argo-table-list__row applications-list__entry applications-list__entry--health-${app.status.health.status} ${
+                                                selectedApp === i ? 'applications-tiles__selected' : ''
+                                            }`}>
+                                            <div className='row' onClick={e => ctx.navigation.goto(`/applications/${app.metadata.name}`, {view: pref.appDetails.view}, {event: e})}>
+                                                <div className={`columns small-12 applications-list__info qe-applications-list-${app.metadata.name}`}>
+                                                    <div className='applications-list__external-link'>
+                                                        <ApplicationURLs urls={app.status.summary.externalURLs} />
                                                     </div>
-                                                </div>
-                                                <div className='row'>
-                                                    <div className='columns small-3' title='Project:'>
-                                                        Project:
+                                                    <div className='row'>
+                                                        <div className='columns small-9'>
+                                                            <i className={'icon argo-icon-' + (app.spec.source.chart != null ? 'helm' : 'git')} />
+                                                            <span className='applications-list__title'>{app.metadata.name}</span>
+                                                        </div>
+                                                        <div className='columns small-3'>
+                                                            <Tooltip content={favlist?.includes(app.metadata.name) ? 'Remove Favourite' : 'Add Favourite'}>
+                                                                <button
+                                                                    onClick={e => {
+                                                                        e.stopPropagation();
+                                                                        favlist?.includes(app.metadata.name)
+                                                                            ? (favlist = favlist.filter(item => item !== app.metadata.name))
+                                                                            : favlist.push(app.metadata.name);
+                                                                        services.viewPreferences.updatePreferences({appList: {...pref.appList, favouritesApplist: favlist}});
+                                                                    }}>
+                                                                    <i
```

Comment:
Can we right justify the star all the way? The easiest way to accomplish this is to wrap the application title and star button in a flexbox (`display: flex`), and set `margin-left` to `auto` for the star.

Reply:
Either right-justified, or a little right-padded next to the application name would be my ask as well.

Reply:
Yes, definitely - trying to understand where this routes get added, so that the icons are placed properly 
![rotue-app](https://user-images.githubusercontent.com/17771352/150097556-5dffd52b-2206-4eb6-99be-601bd19f46f7.png)